packages/ui-scripts/lib/build/examples.js

```diff
@@ -58,7 +58,7 @@ if (args.includes('--watch')) {
   // --no-manager-cache is a workaround for
   // https://github.com/storybookjs/storybook/issues/13200
   // -c: config directory
-  commandArgs = ['-c', '.storybook', '-p', port, '--no-manager-cache']
```

Comment:
the --quiet option removes the "building 54%..." spam from the console. It was outputting so many messages that it was slowing down the build